.circleci/config.yml

```diff
@@ -1,13 +1,28 @@
-# Use the latest 2.1 version of CircleCI pipeline process engine. See: https://circleci.com/docs/2.0/configuration-reference
-version: 2.1
-# Use a package of configuration called an orb.
-orbs:
-  # Declare a dependency on the welcome-orb
-  welcome: circleci/welcome-orb@0.4.1
-# Orchestrate or schedule a set of jobs
-workflows:
-  # Name the workflow "welcome"
-  welcome:
-    # Run the welcome/run job in its own container
-    jobs:
-      - welcome/run
\ No newline at end of file
+version: 2.0
+
+jobs:
+  build:
+    docker:
+      - image: cibuilds/docker:19.03
+    steps:
+      - checkout
+      - setup_docker_php
+      - run:
+          name: "Build Docker Image"
+          command: docker build --file Dockerfile
+      - deploy:
```

Comment:
The deploy step here could possibly conflict with docker hub integration. Please drop this for now.